src/config/constants.ts

```diff
@@ -20,6 +17,15 @@ export const HACKATHON_DATE = 'February 1-2, 2020';
 export const HACKATHON_LOCATION = 'McGill University';
 export const STATIC_SITE = 'https://mchacks.ca';
 
+// Travel information
+export const BUS_DEPARTURE_CITY = 'Toronto';
```

Comment:
Outside the scope of this PR, but we should definitely restructure this eventually to support multiple buses.

Reply:
This would be within the Travel information schema.